canary-checker/docusaurus.config.js

```diff
@@ -142,6 +142,11 @@ export default async function createConfigAsync() {
               position: 'left',
               label: 'Docs',
             },
+            {
+              href: "https://demo.flanksource.com/",
```

Comment:
@moshloop Do we have a running demo for canary checker ?

Reply:
Can you set one up ?